_sass/components/_hero.scss

```diff
@@ -28,6 +28,11 @@
       font-size: 1.25rem; // 20px
     }
   }
+
+  &.partners {
+    background-color: $white;
+    color: $base-font-color;
```

Comment:
I'm not particularly a fan of these variables and would prefer we just use the design system utilities directly.

```suggestion
    background-color: color('white');
    color: color('ink');
```